autoload/neomake/makers/ft/typescript.vim

```diff
@@ -7,8 +7,9 @@ endfunction
 function! neomake#makers#ft#typescript#tsc()
     return {
         \ 'args': [
-            \ '-m', 'commonjs', '--noEmit'
+            \ '--project', getcwd(), '--noEmit'
```

Comment:
If the dir of the file would be enough I think something like `%p:h` could be used.